core/src/commonMain/kotlin/com/ustadmobile/core/controller/VideoContentPresenterCommon.kt

```diff
@@ -38,6 +39,8 @@ abstract class VideoContentPresenterCommon(context: Any, arguments: Map<String,
 
     var timeVideoPlayed = 0L
 
+    var contextRegistration: String? = null
```

Comment:
Use lateinit instead of nullable. onCreate is always going to be called first. This value will never really be null in the lifecycle